packages/TorneloScoresheet/src/style/images.ts

```diff
@@ -26,6 +26,8 @@ import ICON_UNDO_ASSET from '../../assets/images/buttonIcons/icon_undo.svg';
 import ICON_REDO_ASSET from '../../assets/images/buttonIcons/icon_redo.svg';
 import ICON_CLOCK_ASSET from '../../assets/images/buttonIcons/icon_clock.svg';
 import ICON_GEAR_ASSET from '../../assets/images/buttonIcons/icon_gear.svg';
+import ICON_QR_ASSET from '../../assets/images/buttonIcons/icon_qr.svg';
```

Comment:
It might be worth having a chat with Harry about adding images as assets because he mentioned that we should be using material icons where possible and they have one for a qr code. But I'm happy to approve it if you think this way is better.

Reply:
yeh i know harry prefers the googel font icons, but david said their policy is normaly to download the assets and use them as svgs, but he did also say they dont really mind either way, here i used the svg files because i need to use the IconTextButton and i cbf changing the component to work with the material icons haha